packages/tasks/src/dataset-libraries.ts

```diff
@@ -53,6 +53,18 @@ export const DATASET_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/dask/dask",
 		docsUrl: "https://huggingface.co/docs/hub/datasets-dask",
 	},
+	distilabel: {
+		prettyLabel: "Distilabel",
+		repoName: "distilabel",
+		repoUrl: "https://github.com/argilla-io/distilabel",
+		docsUrl: "https://distilabel.argilla.io",
+	},
+	fiftyone: {
+		prettyLabel: "FiftyOne",
+		repoName: "fiftyone",
+		repoUrl: "https://github.com/voxel51/fiftyone",
+		docsUrl: "https://docs.voxel51.com",
+	},
```

Comment:
As requested, it would be great to also include Argilla here 🤗.

```suggestion
	},
        argilla: {
                prettyLabel: "Argilla",
                repoName: "argilla",
                repoUrl: "https://github.com/argilla-io",
                docsUrl: "https://argilla-io.github.io/argilla",
	},
```

Reply:
Perfect, thanks!